pkg/operator/operator_test.go

```diff
@@ -208,6 +208,9 @@ func TestOperatorSyncClusterAPIControllerHealthCheckController(t *testing.T) {
 			},
 		},
 		expectedMachineHealthCheckController: true,
+	}, {
```

Comment:
How does this verify the feature is disable after it's enabled? The operator is initialized from scratch for every test case.

Reply:
indeed, this is duplicating the tc https://github.com/openshift/machine-api-operator/pull/299/files#diff-d5d9cf6e8104e5231787b5cc89fb7b52R198